### ISSUE TITLE
Abnormal circulating Fibulin 1C concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5997]
**Preferred term label:** Abnormal circulating Fibulin 1C concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of fibulin 1C in the blood is outside the normal range., PMID: 31343988

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Fibulin 1C concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of fibulin 1C in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Fibulin 1C concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Fibulins are matrix proteins that do not circulate. I do not think there is any clinical measurement. I am closing this item, but please reopen if there is a pubmed article describing a serum measurement. The HPO does not include measurements that are only done in rare research contexts such as 31343988. @JillMiller
